### PR TITLE
youtube-cleanup: update copyright notice rule

### PR DESCRIPTION
**Pull request by JohnyP36**
In the old youtube layout they changed the source code of the copyright notice. Therefore the rule needs to be changed.

See `https://www.youtube.com/watch?v=WYU4fBQVVOY` 

<details><summary>Screenshot</summary> 

image

</details>

### CODE REVIEW

File: data/filters/youtube-cleanup.yaml [hunk 1]
```diff
@@ -44,7 +44,7 @@ template: |
   www.youtube.com##.ytd-watch-metadata > .top-level-buttons #text:not(:has-text(/[\d]/))
   {{/if}}
   {{#if remove-copyright-notice}}
-  www.youtube.com##[class*="video-secondary-info-renderer"] #always-shown + #collapsible
+    www.youtube.com##[class*="video-secondary-info-renderer"] > #items:has(.ytd-video-description-music-section-renderer)
```
Comment (xvello) [inline data/filters/youtube-cleanup.yaml:47]: Bad identation
```suggestion
  www.youtube.com##[class*="video-secondary-info-renderer"] > #items:has(.ytd-video-description-music-section-renderer)
```

Comment (xvello) [inline data/filters/youtube-cleanup.yaml:47]: You'll need to update the corresponding test, line 80

**Review by xvello — CHANGES_REQUESTED** [review]
Thanks @JohnyP36, two comments on this change.

Also, do you think we should keep the old filter for a while? I'm a bit confused as this is a new version of the old layout? How come you're not on the "new" layout?

**Comment by JohnyP36** [comment]
> Thanks @JohnyP36, two comments on this change.
> 
> Also, do you think we should keep the old filter for a while? I'm a bit confused as this is a new version of the old layout? How come you're not on the "new" layout?

Maybe it's better to also keep the old filter for a while. I have put it back. 

I don't know why I'm still on the old layout and not the "new" layout. Maybe it has to do with the fact that YouTube is still testing it and the new layout is rolling out in stages.

**Review by xvello — APPROVED** [review]
Thanks a lot!

**Pull request merged**